.travis.yml

```diff
@@ -2,8 +2,6 @@ language: python
 sudo: false
```

Comment:
```suggestion
sudo: false
```

This is deprecated stuff in TravisCI without purpose any more.

Reply:
Oh sorry, I meant to remove this line entirely, but instead I removed one space in the github commit suggestion